### PR TITLE
enable tests for Qt6

### PR DESCRIPTION
**Pull request by 3nids**

### CODE REVIEW

**Comment by 3nids** [comment]
79% tests passed, 45 tests failed out of 213

Total Test time (real) = 413.81 sec

The following tests FAILED:
	  8 - test_core_authconfig (Failed)
	 15 - test_core_callout (Failed)
	 21 - test_core_compositionconverter (Failed)
	 28 - test_core_datadefinedsizelegend (Failed)
	 33 - test_core_dxfexport (Failed)
	 35 - test_core_expression (Failed)
	 38 - test_core_field (Failed)
	 41 - test_core_fontmarker (Failed)
	 46 - test_core_geometry (Failed)
	 61 - test_core_labelingengine (Failed)
	 62 - test_core_layertree (Failed)
	 63 - test_core_layout (Failed)
	 68 - test_core_layouthtml (Timeout)
	 71 - test_core_layoutlabel (Failed)
	 72 - test_core_layoutmanualtable (Failed)
	 73 - test_core_layoutmap (Failed)
	 74 - test_core_layoutmapgrid (Failed)
	 75 - test_core_layoutmapoverview (Failed)
	 77 - test_core_layoutmultiframe (Failed)
	 80 - test_core_layoutpicture (Failed)
	 82 - test_core_layoutscalebar (Failed)
	 84 - test_core_layouttable (Subprocess aborted)
	 87 - test_core_legendrenderer (Failed)
	 89 - test_core_mapdevicepixelratio (Failed)
	 93 - test_core_maprendererjob (Failed)
	 94 - test_core_maprotation (Failed)
	103 - test_core_meshlayer (Failed)
	105 - test_core_meshlayerrenderer (Failed)
	107 - test_core_networkaccessmanager (Timeout)
	111 - test_core_ogcutils (Failed)
	118 - test_core_pallabeling (Failed)
	119 - test_core_point (Failed)
	123 - test_core_pointpatternfillsymbol (Failed)
	125 - test_core_project (Subprocess aborted)
	126 - test_core_projectstorage (Failed)
	137 - test_core_rasterlayer (Subprocess aborted)
	139 - test_core_rastermarker (Failed)
	159 - test_core_style (Failed)
	167 - test_core_tiledownloadmanager (Subprocess aborted)
	179 - test_core_vectortilelayer (Failed)
	184 - test_core_coordinatereferencesystem (Failed)
	187 - test_analysis_processingalgs (Failed)
	188 - test_analysis_zonalstatistics (Subprocess aborted)
	195 - test_analysis_meshcontours (Failed)
	199 - test_geometry_checker_geometrychecks (Failed)

File: .ci/test_blocklist_qt6.txt [hunk 1]
```diff
@@ -0,0 +1,71 @@
+# Qt6 blacklist
```
Comment (nyalldawson) [inline .ci/test_blocklist_qt6.txt:1]:
```suggestion
# Qt6 blocklist
```

File: .ci/test_blocklist_qt6.txt [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+test_core_style
+test_core_tiledownloadmanager
+test_core_vectortilelayer
+test_core_coordinatereferencesystem
+test_analysis_processingalgs
+test_analysis_zonalstatistics
+test_analysis_meshcontours
+test_geometry_checker_geometrychecks
+
+# black list
```
Comment (nyalldawson) [inline .ci/test_blocklist_qt6.txt:48]:
```suggestion
# block list
```

**Review by nyalldawson — APPROVED** [review]
Woohoo! :partying_face:

**Pull request merged**